litmus-portal/frontend/src/components/Button/ButtonOutline/styles.ts

```diff
@@ -5,7 +5,7 @@ const useStyles = makeStyles((theme) => ({
     display: 'flex',
     flexDirection: 'row',
     minWidth: '4.375rem',
-    height: '2.8125rem',
+    height: '2.2.6rem',
```

Comment:
Can you change this height to 2.6 or something rather than 2.2.6!

Reply:
For this Button is height is getting unbalance. Should i keep 2.2.6?
I did rest of the changes